src/node_api.cc

```diff
@@ -1618,6 +1637,30 @@ napi_status napi_get_value_string_length(napi_env e,
   return GET_RETURN_STATUS();
 }
 
+// Copies a JavaScript string into a LATIN-1 string buffer. The result is the
+// number
```

Comment:
Can you clean the spacing of this comment, update it to indicate how it works when buf is NULL, and remove any non-UTF-8 whitespace characters (IIRC, the comments near other string functions had such characters)?